backend/src/main/java/com/example/skicurort/exception/NoIdException.java

```diff
@@ -0,0 +1,9 @@
+package com.example.skicurort.exception;
+
+public class NoIdException extends RuntimeException {
+
+  public NoIdException(Long id) {
+
+    super("Could not find object with id: " + id);
+  }
+}
```

Comment:
N koniec - nie powinniśmy merdżować kodu, który nie ma testów.

Reply:
to prawda, trochę chciałem cokolwiek puścić by sprawdzić czy z merdżowaniem nie bedzie problemu

Reply:
ok ale ryzykujesz, że ktoś Ci to popsuje jak  nie będzie otestowane :P

Reply:
Wiem wiem ;) , dlatego teraz siedzę nad tymi testami ale chyba mój mozg dziś tego nie ogarnia a może nie dziś;)